lib/json_api_resource/queryable.rb

```diff
@@ -15,28 +15,35 @@ def find(id)
         results = self.client_klass.find(id).map! do |result|
           self.new(:client => result)
         end
-        results.size == 1 ? single_result(results) : results
+        results = results.size == 1 ? single_result(results) : results
+        package_result(results)
```

Comment:
We wanted to make sure we can always use symbols application-side, but we decided to go with an indifferent hash in json_api_client instead of this.  This is removed now.